🌐 Add Polish translation for `docs/pl/docs/help-fastapi.md` (#10121)
fastapi/fastapi: docs/pl/docs/help-fastapi.md

Jeśli wybierzesz "Obserwuj" zamiast "Tylko wydania", otrzymasz powiadomienia, gdy ktoś utworzy nowy problem lub pytanie. Możesz również określić, że chcesz być powiadamiany tylko o nowych problemach, dyskusjach, PR-ach itp.

Następnie możesz spróbować pomóc im rozwiązać te pytania.

## Zadawaj pytania

Możesz <a href="https://github.com/tiangolo/fastapi/discussions/new?category=questions" class="external-link" target="_blank">utworzyć nowe pytanie</a> w repozytorium na GitHubie, na przykład aby:

* Zadać **pytanie** lub zapytać o **problem**.
* Zaproponować nową **funkcję**.

**Uwaga**: jeśli to zrobisz, poproszę Cię również o pomoc innym. 😉

## Przeglądaj Pull Requesty

Możesz pomóc mi w przeglądaniu żądań ściągnięcia od innych.

Ponownie, proszę, postaraj się być jak najbardziej życzliwy. 🤗

---

Oto, co warto mieć na uwadze i jak przeglądać pull request:

### Zrozum problem

* Najpierw upewnij się, że **rozumiesz problem**, który próbuje rozwiązać pull request. Może on mieć dłuższą dyskusję w Dyskusji na GitHubie lub w problemie.

> * Jest też duża szansa, że pull request nie jest tak naprawdę potrzebny, ponieważ problem można rozwiązać w **inny sposób**. Wtedy możesz to zasugerować lub o to zapytać.

Reviewer: ```suggestion
* Jest też duża szansa, że pull request nie jest konieczny, ponieważ problem można rozwiązać w **inny sposób**. Wtedy możesz to zasugerować lub o to zapytać.
```

### Nie martw się stylem

> * Nie martw się zbytnio rzeczami takimi jak style wiadomości commitów, scalę i połączę je ręcznie, dostosowując commit.

Reviewer: ```suggestion
* Nie przejmuj się zbytnio rzeczami takimi jak style wiadomości commitów, przy wcielaniu pull requesta łączę commity i modyfikuję opis sumarycznego commita ręcznie.
```

> * Nie martw się również zasadami stylu, istnieją już zautomatyzowane narzędzia sprawdzające to.

Reviewer: ```suggestion
* Nie przejmuj się również stylem kodu, automatyczne narzędzia w repozytorium sprawdzają to samodzielnie.
```

> A jeśli istnieje jakaś inna potrzeba dotycząca stylu lub spójności, poproszę bezpośrednio o to, lub dodam commity z potrzebnymi zmianami.

Reviewer: ```suggestion
A jeśli istnieje jakaś konkretna potrzeba dotycząca stylu lub spójności, sam poproszę o zmiany lub dodam commity z takimi zmianami.
```

### Sprawdź kod

> * Sprawdź i przeczytaj kod, zobacz, czy ma sens, **uruchom go lokalnie** i zobacz, czy faktycznie rozwiązuje problem.

Reviewer: ```suggestion
* Przeczytaj kod, zastanów się czy ma sens, **uruchom go lokalnie** i potwierdź czy faktycznie rozwiązuje problem.
```

> * Następnie **skomentuj**, mówiąc, że to zrobiłeś, dzięki temu będę wiedział, że naprawdę to sprawdziłeś.

Reviewer: ```suggestion
* Następnie dodaj **komentarz** z informacją o tym, że sprawdziłeś kod, dzięki temu będę miał pewność, że faktycznie go sprawdziłeś.
```

!!! info

>     Niestety, nie mogę po prostu ufać PR-om, które mają kilka zatwierdzeń.

Reviewer: ```suggestion
    Niestety, nie mogę ślepo ufać PR-om, nawet jeśli mają kilka zatwierdzeń.
```

>     Kilka razy zdarzyło się, że są PR-y z 3, 5 lub więcej zatwierdzeniami, prawdopodobnie dlatego, że opis jest interesujący, ale gdy sprawdzam PR-y, są one w rzeczywistości uszkodzone, mają błąd lub nie rozwiązują problemu, który twierdzą, że rozwiązują. 😅

Reviewer: ```suggestion
    Kilka razy zdarzyło się, że PR-y miały 3, 5 lub więcej zatwierdzeń (prawdopodobnie dlatego, że opis obiecuje rozwiązanie ważnego problemu), ale gdy sam sprawdziłem danego PR-a, okazał się być zbugowany lub nie rozwiązywał problemu, który rzekomo miał rozwiązywać. 😅
```

>     Dlatego naprawdę ważne jest, abyś faktycznie przeczytał i uruchomił kod oraz dał mi znać w komentarzach, że to zrobiłeś. 🤓

Reviewer: ```suggestion
    Dlatego tak ważne jest, abyś faktycznie przeczytał i uruchomił kod oraz napisał w komentarzu, że to zrobiłeś. 🤓
```

> * Jeśli PR można uprościć w jakiś sposób, możesz o to poprosić, ale nie ma potrzeby być zbyt wybrednym, może być wiele subiektywnych punktów widzenia (a ja też będę miał swoje 🙈), więc lepiej, jeśli skupisz się na podstawowych rzeczach.

Reviewer: ```suggestion
* Jeśli PR można uprościć w jakiś sposób, możesz o to poprosić, ale nie ma potrzeby być zbyt wybrednym, może być wiele subiektywnych punktów widzenia (a ja też będę miał swój 🙈), więc lepiej żebyś skupił się na kluczowych rzeczach.
```

### Testy

* Pomóż mi sprawdzić, czy PR ma **testy**.

* Sprawdź, czy testy **nie przechodzą** przed PR. 🚨

* Następnie sprawdź, czy testy **przechodzą** po PR. ✅

* Wiele PR-ów nie ma testów, możesz **przypomnieć** im o dodaniu testów, a nawet **zaproponować** samemu jakieś testy. To jedna z rzeczy, które pochłaniają najwięcej czasu i możesz w tym bardzo pomóc.

* Następnie skomentuj również to, czego spróbowałeś, wtedy będę wiedział, że to sprawdziłeś. 🤓

## Utwórz Pull Request

Możesz [wnieść wkład](contributing.md){.internal-link target=_blank} do kodu źródłowego za pomocą Pull Requestu, na przykład:

* Naprawić literówkę, którą znalazłeś w dokumentacji.
* Podzielić się artykułem, filmem lub podcastem, który stworzyłeś lub znalazłeś na temat FastAPI, <a href="https://github.com/tiangolo/fastapi/edit/master/docs/en/data/external_links.yml" class="external-link" target="_blank">edytując ten plik</a>.
    * Upewnij się, że dodajesz swój link na początku odpowiedniej sekcji.
* Pomóc w [tłumaczeniu dokumentacji](contributing.md#translations){.internal-link target=_blank} na Twój język.

>     * Możesz również pomóc w recenzji tłumaczeń stworzonych przez innych.

Reviewer: ```suggestion
    * Możesz również pomóc w weryfikacji tłumaczeń stworzonych przez innych.
```

* Zaproponować nowe sekcje dokumentacji.
* Naprawić istniejący problem/błąd.
    * Upewnij się, że dodajesz testy.
* Dodać nową funkcję.
    * Upewnij się, że dodajesz testy.
    * Upewnij się, że dodajesz dokumentację, jeśli jest to istotne.

## Pomóż w utrzymaniu FastAPI

Pomóż mi utrzymać **FastAPI**! 🤓

Jest wiele pracy do zrobienia, a w większości przypadków **TY** możesz to zrobić.

Główne zadania, które możesz wykonać teraz to:

> * [Pomóc innym z pytaniami na GitHubie](#help-others-with-questions-in-github){.internal-link target=_blank} (zobacz powyższą sekcję).

Reviewer: ```suggestion
* [Pomóc innym z pytaniami na GitHubie](#help-others-with-questions-in-github){.internal-link target=_blank} (zobacz sekcję powyżej).
```

> * [Oceniać Pull Requesty](#review-pull-requests){.internal-link target=_blank} (zobacz powyższą sekcję).

Reviewer: ```suggestion
* [Oceniać Pull Requesty](#review-pull-requests){.internal-link target=_blank} (zobacz sekcję powyżej).
```

Te dwie czynności **zajmują najwięcej czasu**. To główna praca związana z utrzymaniem FastAPI.

> Jeśli możesz mi w tym pomóc, **pomagasz mi utrzymać FastAPI** i upewnić się, że nadal **rozwija się szybciej i lepiej**. 🚀

Reviewer: ```suggestion
Jeśli możesz mi w tym pomóc, **pomożesz mi utrzymać FastAPI** i zapewnisz że będzie **rozwijać się szybciej i lepiej**. 🚀
```

## Dołącz do czatu

Dołącz do 👥 <a href="https://discord.gg/VQjSZaeJmf" class="external-link" target="_blank">serwera czatu na Discordzie</a> 👥 i spędzaj czas z innymi w społeczności FastAPI.

!!! wskazówka
    Jeśli masz pytania, zadaj je w <a href="https://github.com/tiangolo/fastapi/discussions/new?category=questions" class="external-link" target="_blank">Dyskusjach na GitHubie</a>, jest dużo większa szansa, że otrzymasz pomoc od [Ekspertów FastAPI](fastapi-people.md#experts){.internal-link target=_blank}.

    Używaj czatu tylko do innych ogólnych rozmów.

Istnieje również poprzedni <a href="https://gitter.im/tiangolo/fastapi" class="external-link" target="_blank">czat na Gitter</a>, ale ponieważ nie ma tam kanałów i zaawansowanych funkcji, rozmowy są trudniejsze, dlatego teraz zalecany jest Discord.

### Nie zadawaj pytań na czacie

Miej na uwadze, że ponieważ czaty pozwalają na bardziej "swobodną rozmowę", łatwo jest zadawać pytania, które są zbyt ogólne i trudniejsze do odpowiedzi, więc możesz nie otrzymać odpowiedzi.

> Na GitHubie szablon poprowadzi Cię do napisania odpowiedniego pytania, dzięki czemu łatwiej uzyskasz dobrą odpowiedź, a nawet rozwiążesz problem samodzielnie, zanim zapytasz. Pondato na GitHubie mogę się upewnić, że zawsze odpowiadam na wszystko, nawet jeśli zajmuje to trochę czasu. Osobiście nie mogę tego zrobić z systemami czatu. 😅

Reviewer: ```suggestion
Na GitHubie szablon poprowadzi Cię do napisania odpowiedniego pytania, dzięki czemu łatwiej uzyskasz dobrą odpowiedź, a nawet rozwiążesz problem samodzielnie, zanim zapytasz. Ponadto na GitHubie mogę się upewnić, że zawsze odpowiadam na wszystko, nawet jeśli zajmuje to trochę czasu. Osobiście nie mogę tego zrobić z systemami czatu. 😅
```

Rozmowy w systemach czatu nie są też tak łatwo przeszukiwalne jak na GitHubie, więc pytania i odpowiedzi mogą zaginąć w rozmowie. A tylko te na GitHubie liczą się do zostania [Ekspertem FastAPI](fastapi-people.md#experts){.internal-link target=_blank}, więc najprawdopodobniej otrzymasz więcej uwagi na GitHubie.

> Z drugiej strony w systemach czatu jest tysiące użytkowników, więc jest duża szansa, że znajdziesz tam kogoś do rozmowy, prawie cały czas. 😄

Reviewer 2: ```suggestion
Z drugiej strony w systemach czatu są tysiące użytkowników, więc jest duża szansa, że znajdziesz tam kogoś do rozmowy, prawie w każdej chwili. 😄
```
